news/2024/2024-01-07-featured-artist-track-updates-winter-2024.md

```diff
@@ -0,0 +1,254 @@
+
+- **ColBreakz - 100.000**
+- **ColBreakz - 75.000**
+- **ColBreakz - Down Caves**
+- **ColBreakz - Fake Vibes**
+- **ColBreakz - Geometry Hazard**
+- **ColBreakz - Hello**
+- **ColBreakz - Lost Star**
+- **ColBreakz - Pixel World**
+- **ColBreakz - The One**
+- **ColBreakz - You & Me**
+
+### ColBreakz - 100.000
+
+<audio controls>
+    <source src="LINK" type="audio/mpeg">
+</audio>
+
+### ColBreakz - Lost Star
+
+<audio controls>
+    <source src="LINK" type="audio/mpeg">
+</audio>
+
+### ColBreakz - Geometry Hazard
+
+<audio controls>
+    <source src="LINK" type="audio/mpeg">
+</audio>
+
+## [cygnus](https://osu.ppy.sh/beatmaps/artists/359)
+
+[![](https://assets.ppy.sh/artists/359/header.jpg)](https://osu.ppy.sh/beatmaps/artists/359)
+
+Born and raised on osu!, [**cygnus**](https://osu.ppy.sh/beatmaps/artists/359) is fully aware of what it means to grind. Since his Featured Artist announcement in May 2023, he created **22** new songs (and provided all of them to the newest generation of circle clickers!):
+
+- **cygnus - Aishuu Ningyo**
+- **cygnus - Anvilia**
+- **cygnus - Butterfly Killer**
+- **cygnus - Dark Ring System**
+- **cygnus - Darkness Parade**
+- **cygnus - Dec. 31st, 1999**
+- **cygnus - eternal punishment**
+- **cygnus - Gesshoku Butou**
+- **cygnus - Jyoubachi no Danmatsuma**
+- **cygnus - LuNa99998**
+- **cygnus - Moon Flower**
+- **cygnus - Mortem Oppetere**
+- **cygnus - Mr. Psychopath**
+- **cygnus - Omaju**
+- **cygnus - Parasite Circus**
+- **cygnus - Shinshoku**
+- **cygnus - Silver Mutation**
+- **cygnus - Silver Orbit**
+- **cygnus - St. Ophelia**
+- **cygnus - Sun Flower**
+- **cygnus - Tasogare no Megami**
+- **cygnus - Twilight Ceremony**
+
+### cygnus - Silver Mutation
+
+Try out [this osu! marathon](https://osu.ppy.sh/beatmapsets/2067206) hosted by [the man behind the music](https://osu.ppy.sh/beatmapsets/2067206)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/359/Omaju/cygnus - Silver Mutation.mp3" type="audio/mpeg">
+</audio>
+
+### cygnus - Anvilia
+
+Or try [this wiggle-stream masterpiece](https://osu.ppy.sh/beatmapsets/2056846) hosted by [LMT](https://osu.ppy.sh/users/7262798)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/359/The Evolution of War/cygnus_-_Anvilia.mp3" type="audio/mpeg">
+</audio>
+
+### cygnus - Omaju
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/359/Omaju/cygnus - Omaju.mp3" type="audio/mpeg">
+</audio>
+
+## [HyuN](https://osu.ppy.sh/beatmaps/artists/33)
+
+[![](https://assets.ppy.sh/artists/33/header.jpg)](https://osu.ppy.sh/beatmaps/artists/33)
+
+One of osu!'s first Featured Artists makes a return, this time with just *one* new track. You don't have much choice here, so start mapping already:
+
+### HyuN - iRELLiA
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/33/Songs/HyuN_-_iRELLiA.mp3" type="audio/mpeg">
+</audio>
+
+## [SAMString](https://osu.ppy.sh/beatmaps/artists/273)
```

Comment:
there are 4 new tracks but you've listed one

![image](https://github.com/ppy/osu-wiki/assets/36758269/e651a5d0-efc4-4af1-953a-867450b9a747)

Reply:
3 are old, but the artist gave album art for them, so they show up as new. i can't un-newify them :(